lint/rule_target_job_instance.go

```diff
@@ -12,7 +12,7 @@ func newTargetRequiredMatcherRule(matcher string) *TargetRuleFunc {
 		name:        fmt.Sprintf("target-%s-rule", matcher),
 		description: fmt.Sprintf("Checks that every PromQL query has a %s matcher.", matcher),
 		fn: func(d Dashboard, p Panel, t Target) Result {
-			// TODO: The RuleSet should be responsible for routing rule checks based on their query type (prometheus, loki, mysql, etc)
```

Comment:
I presume that the TODOs were removed as a result of the updated linting? Can we override that rule?

All of the TODOs which were removed are still not complete, and.. I've been a poor maintainer in that they aren't tracked as issues either 🤦

Reply:
yes, we can override that rule - I'll do that